### ISSUE TITLE
feature: Ability to output in json for `ibc-setup`

### ISSUE THREAD

**Issue by Anmol1696** [confio/ts-relayer#251]
## Overview
Inorder to use ts-relayer as a standalone relayer like hermes, it might make sense to have additional flag of `--output <json | yaml | default>` for all the queries with `ibc-setup *`.

## Why
To be able to use `jq` with `ibc-setup` directly in bash instead of having to create a custom wrapper around the relayer. This will reduce the overhead of running the relayer with the current opperators as well (moving ts-relayer to more production usecases).

Will be a nice to have

**Comment by ethanfrey** [comment]
I agree, this would be a nice improvement. I don't have time for this now, but happy to review and approve a pr

**Comment by Anmol1696** [comment]
Cool, will try to get this done.

**Comment by ethanfrey** [comment]
Hmmm... looks like there is no abstraction here, [just some console.log stuff](https://github.com/confio/ts-relayer/blob/main/src/binary/ibc-setup/commands/channel.ts#L145-L153). Happy if you make all those lines in `inc-setup` if statements for two outputs. No need to auto-generate them from some meta-format.

Also, note the verbose/debug logs will always be the same. You can just turn down log level for scripts (via `-q`), so you only get the final (JSON) output.
